package.json

```diff
@@ -94,9 +98,9 @@
     "clean:build": "shx rm -rf .cache",
     "commit": "git-cz",
     "generate": "yarn plop",
-    "lint:md": "yarn eslint --ext md ./packages/patternfly-4 --config ./.eslintrc_md.json",
+    "lint:md": "yarn eslintpackages/patternfly-4 --ext md --config .eslintrc-md.json",
```

Comment:
space missing after `eslint`?

Reply:
Yeah, let's make a followup PR to fix that and turn it on in CI.